readme.txt

```diff
@@ -2,8 +2,8 @@
 Contributors: mboynes,alleyinteractive
 Tags: options, importer, exporter, export, import, migrate, settings, wp_options
 Requires at least: 3.8
-Tested up to: 3.9
-Stable tag: 7
+Tested up to: 5.4
```

Comment:
Can we preemptively test against 5.5 since it comes out in a few days and avoid the need to update this again?